src/shared/file_op.c

```diff
@@ -1900,7 +1901,14 @@ const char *getuname()
                             snprintf(__wp, 63, " [Ver: %d.%d]", (unsigned int)winMajor, (unsigned int)winMinor);
                         }
                         else {
-                            snprintf(__wp, 63, " [Ver: %d.%d.%s]", (unsigned int)winMajor, (unsigned int)winMinor, wincomp);
+                            dwCount = size;
+                            dwRet = RegQueryValueEx(RegistryKey, TEXT("UBR"), NULL, &type, (LPBYTE)&buildRevision, &dwCount);
+                            if (dwRet != ERROR_SUCCESS) {
+                                snprintf(__wp, 63, " [Ver: %d.%d.%s]", (unsigned int)winMajor, (unsigned int)winMinor, wincomp);
```

Comment:
This is not specific to the PR but I figured I should at least mention it: we don't need to use the `63` integer literal here, we can just use `sizeof _wp` or `sizeof(_wp)` because according to the snprintf documentation ([MSVC](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/snprintf-snprintf-snprintf-l-snwprintf-snwprintf-l?view=msvc-170), [POSIX](https://www.unix.com/man-page/POSIX/3posix/snprintf/)), `snprint` will not write past byte `n - 1` (where `n` is the second parameter to `snprintf`)

Reply:
I will change right now. Thank you Ramiro.